packages/tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,312 @@
+import React, { useState, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import Tab, { TabProps } from './Tab';
+import TabTitle from './TabTitle';
+
+const listStyle = css`
+  list-style: none;
+  padding: 0px;
+  margin-top: 20px;
+  margin-bottom: 5px;
+`;
+
+const listTitle = css`
```

Comment:
Shouldn't these styles be baked into the tab title component instead of be in this file?

Reply:
👍